kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/resources/operator/KroxyliciousOperatorBundleInstaller.java

```diff
@@ -0,0 +1,402 @@
+        KroxyliciousOperatorBundleInstaller otherInstallation = (KroxyliciousOperatorBundleInstaller) other;
+
+        return operationTimeout == otherInstallation.operationTimeout &&
+                reconciliationInterval == otherInstallation.reconciliationInterval &&
+                Objects.equals(cluster, KroxyliciousOperatorBundleInstaller.cluster) &&
+                Objects.equals(kroxyliciousOperatorName, otherInstallation.kroxyliciousOperatorName) &&
+                Objects.equals(namespaceInstallTo, otherInstallation.namespaceInstallTo) &&
+                Objects.equals(namespaceToWatch, otherInstallation.namespaceToWatch) &&
+                Objects.equals(bindingsNamespaces, otherInstallation.bindingsNamespaces) &&
+                Objects.equals(extraLabels, otherInstallation.extraLabels);
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(cluster, extensionContext,
+                kroxyliciousOperatorName, namespaceInstallTo, namespaceToWatch, bindingsNamespaces, operationTimeout, extraLabels);
+    }
+
+    @Override
+    public String toString() {
+        return "KroxyliciousOperatorBundleInstaller{" +
+                "cluster=" + KroxyliciousOperatorBundleInstaller.cluster +
+                ", extensionContext=" + extensionContext +
+                ", kroxyliciousOperatorName='" + kroxyliciousOperatorName + '\'' +
+                ", namespaceInstallTo='" + namespaceInstallTo + '\'' +
+                ", namespaceToWatch='" + namespaceToWatch + '\'' +
+                ", bindingsNamespaces=" + bindingsNamespaces +
+                ", operationTimeout=" + operationTimeout +
+                ", reconciliationInterval=" + reconciliationInterval +
+                ", extraLabels=" + extraLabels +
+                ", testClassName='" + testClassName + '\'' +
+                ", testMethodName='" + testMethodName + '\'' +
+                '}';
+    }
+
+    private void setTestClassNameAndTestMethodName() {
+        this.testClassName = this.extensionContext.getRequiredTestClass() != null ? this.extensionContext.getRequiredTestClass().getName() : "";
+
+        try {
+            if (this.extensionContext.getRequiredTestMethod() != null) {
+                this.testMethodName = this.extensionContext.getRequiredTestMethod().getName();
+            }
+        }
+        catch (PreconditionViolationException e) {
+            LOGGER.debug("Test method is not present: {}\n{}", e.getMessage(), e.getCause());
+            // getRequiredTestMethod() is not present, in @BeforeAll scope so we're avoiding PreconditionViolationException exception
+            this.testMethodName = "";
+        }
+    }
+
+    @Override
+    public void install() {
+        LOGGER.info("Install Kroxylicious Operator via Yaml bundle in namespace {}", namespaceInstallTo);
+
+        setTestClassNameAndTestMethodName();
+
+        prepareEnvForOperator(namespaceInstallTo, bindingsNamespaces);
+
+        // 03.Deployment
```

Comment:
What's going on here?  I'm understanding that you need to specialcase the Deployment, but what exactly needs to change?